### PR TITLE
[AG-1557] Fixes ModelAD `File size must be at least one byte` Error

### PR DESCRIPTION
**Pull request by BWMac**
**Problem:**
With recent changes made in ADT adding support for ModelAD data processing, we encountered an [error](https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/agora-project/watch/26w3vkx1g3KsQr) in processing when using the `platform` flag. This is caused because despite the fact that there is no GX data validation to run for ModelAD at the moment, the [ADTGXReporter](https://github.com/Sage-Bionetworks/agora-data-tools/blob/7f8eda19ecd249d61959434ef611d5c1c291174a/src/agoradatatools/reporter.py#L73) was still trying to update the GX report table. This caused an empty list of records to be passed to `syn.store` [here](https://github.com/Sage-Bionetworks/agora-data-tools/blob/7f8eda19ecd249d61959434ef611d5c1c291174a/src/agoradatatools/reporter.py#L124), causing the reported error.

**Solution:**
I added an additional condition to `ADTGXReporter.update_table` so that the update attempt is not made if there are no new records to update in the first place.

**Notes:**
- I added a new unit test to cover the new condition
- I went ahead and updated the descriptions for the CLI arguments and options to clarify how to use them and updated the README to tell users to use the `adt --help` command to learn how to use the package.

### CODE REVIEW

**Review by JessterB — APPROVED** [review]
lgtm, thanks Brad!

**Review by thomasyu888 — APPROVED** [review]
🔥 LGTM!  Please create a release after this is merged

**Pull request merged**
